hr_leave_type_code/views/hr_leave_type.xml

```diff
@@ -38,4 +38,17 @@
         </field>
     </record>
 
+    <record id="view_holiday_status_normal_tree" model="ir.ui.view">
+        <field name="name">hr.leave.type.code</field>
+        <field name="model">hr.leave.type</field>
+        <field name="inherit_id" ref="hr_holidays.view_holiday_status_normal_tree" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='display_name']" position="before">
+                <field name="code" optional="hide" />
+                <field name="name" optional="hide" />
```

Comment:
```suggestion
```
IMO this field is not necessary, the `hr_holidays.view_holiday_status_normal_tree` view already adds the `display_name` field.